legend-engine-testable/src/main/test/java/org/finos/legend/engine/test/runner/shared/connection/TestTestConnectionBuilder.java

```diff
@@ -12,7 +12,7 @@
 //  See the License for the specific language governing permissions and
 //  limitations under the License.
 
-package org.finos.legend.engine.testable.service.connection;
+package org.finos.legend.engine.test.runner.shared.connection;
```

Comment:
Can we move this to org.finos.legend.engine.testable.connection

Reply:
seems like the old change, the package is different in the actual changes